tasks.py

```diff
@@ -85,3 +86,29 @@ def set_optimade_ver(_, ver=""):
     update_file("INSTALL.md", (r"/v[0-9]+(\.[0-9]+){2}", f"/v{version}"))
 
     print(f"Bumped OPTiMaDe version to {ver}")
+
+
+@task
+def parse_spec_for_filters(_):
```

Comment:
Nice - a bit vulnerable, but oh well.
But I noticed there are no contingencies for OPTIONAL. Did you make that file manually then?

Reply:
If the spec github gets taken over then we have bigger problems!

The optional file is made manually, and is used to exclude values in filters.txt, until the spec is parseable. I explain the procedure a bit here:
https://github.com/Materials-Consortia/optimade-python-tools/blob/f3875574df2c51f86b09d9824efddc9f0f65c744/optimade/validator/data/__init__.py#L1

Reply:
Great, thanks! :)